### DISCUSSION TITLE
Brand New Loupedeck plus and Commandpost Question

### DISCUSSION THREAD

**Discussion by tonym260** [CommandPost/CommandPost#3301] [General]
Hello, I just got a Loupedeck plus that I'm going to use with command post and FCPX, On1 And Color Finale. I know with On1 and Color Finale I will need to make all the commands and presets manually but, I downloaded the FCPX with CP profile but does that only work with CT? When I try to launch with my plus hardware, it says it can't load the preset. So do I have to do that one manualy as well? Or is there a way to load the presets for CT into the Loupedeck plus or am I always having to start from scratch? And, are there any other custom presets that will work with plus?

**Comment by latenitefilms** [comment]
Generally speaking, most people use CommandPost directly, instead of using the Loupedeck Plugin and the official Loupedeck App.

There's no default profile deliberately - everyone customises it to their own needs, with their own specific effects, transitions, generators, etc.
